src/modify.c

```diff
@@ -23,7 +26,6 @@ void modify() {
     int count = 0; /* checks for error code */
     int empid; /* Employee id */
     char tname[40]; /* Temporary name */

```

Comment:
Do not use hard coded value

Reply:
This is variable initialization, Count is used in many places. So to be on safer side, the variable initialized the moment it is declared.